db/re/item_db_equip.yml

```diff
@@ -67485,7 +67485,7 @@ Body:
     Refineable: true
     View: 963
     Script: |
-      bonus bMagicHPGainValue,100; bonus bMagicSPGainValue,100;
+      bonus bMatk,20; .@r = getrefine(); bonus bMatk,5*.@r; bonus bMagicHPGainValue,100; bonus bSPLossRate,50,5000;
```

Comment:
Separate calls to any  `bonus` function actually override the previous call, so to combine the bonus the final value needs to be calculated and provided in a single call.
Example:
`bonus bMatk, 20+(getrefine()*5)`;

Reply:
I think the loss is also meant to HP rather than SP: https://www.divine-pride.net/database/item/18848/lush-rose-1

Reply:
`bonus bMatk,20+(5*getrefine()); bonus bMagicHPGainValue,100; bonus bSPLossRate,50,5000;`